sql/src/test/java/org/apache/druid/sql/calcite/util/TestServerInventoryView.java

```diff
@@ -51,8 +51,8 @@
       0
   );
   private static final DruidServerMetadata DUMMY_SERVER_REALTIME = new DruidServerMetadata(
-      "dummy",
-      "dummy",
+      "dummy2",
```

Comment:
oh, i should have made it clear. I added segment `test2` in `SystemSchemaTest.java` to `realtimeSegments`, this change is to make the two server names distinct (realtime and historical server type), so that i get 2 replicas for segment `test2` in the unit test.

Reply:
Thanks for explanation :+1: